src/test/pages/Login.page.ts

```diff
@@ -23,6 +24,13 @@ export class LoginPage {
     public async clickLoginBtn() {
         const ele = await this.page.locator("button[type='submit']");
```

Comment:
could you please add locators as a class fields as it is done in MembersPage and not use hardcoded locators in methods?

Reply:
I tried to do this here on the Login page, but i get the error that property page is used before initalization, an error which to this time i was not able to solve